components/app_bar/app_bar.tsx

```diff
@@ -82,6 +82,7 @@ export default function AppBar() {
                     padding-top: 16px;
                     border-left: solid 1px rgba(var(--center-channel-color-rgb), 0.12);
                     background-color: rgba(var(--center-channel-color-rgb), 0.04);
+                    overflow-y: scroll;
```

Comment:
The reason this `overflow-y: scroll` is shifting them left is because of the additional scrollbar it's introducing. (Looks like `.app-bar`'s scrollbar has a `scrollbar-width: none;`)